cabby/data/wikidata/query.py

```diff
@@ -0,0 +1,197 @@
+            wikibase:language "[AUTO_LANGUAGE],en". } SERVICE wikibase:box
+            {?place wdt:P625 ?location . bd:serviceParam wikibase:cornerWest
+            "Point(-74.013,40.752)"^^geo:wktLiteral . bd:serviceParam
+            wikibase:cornerEast "Point(-73.959,40.76)"^^geo:wktLiteral .
+            }
+            }
+            UNION
+            {
+            ?place wdt:P31 ?instance. SERVICE wikibase:label { bd:serviceParam
+            wikibase:language "[AUTO_LANGUAGE],en". } SERVICE wikibase:box
+            {?place wdt:P625 ?location . bd:serviceParam wikibase:cornerWest
+            "Point(-74.01,40.76)"^^geo:wktLiteral . bd:serviceParam
+            wikibase:cornerEast "Point(-73.952,40.7645)"^^geo:wktLiteral .
+            }
+            }
+            UNION
+            {
+            ?place wdt:P31 ?instance. SERVICE wikibase:label { bd:serviceParam
+            wikibase:language "[AUTO_LANGUAGE],en". } SERVICE wikibase:box
+            {?place wdt:P625 ?location . bd:serviceParam wikibase:cornerWest
+            "Point(-74,40.7645)"^^geo:wktLiteral . bd:serviceParam
+            wikibase:cornerEast "Point(-73.946,40.772)"^^geo:wktLiteral .
+            }
+            }
+            }
+            }
+                FILTER (?instance  not in (wd:Q34442,wd:Q12042110,wd:Q124757,wd:Q79007,wd:Q18340514,wd:Q537127,wd:Q1311958,wd:Q124757, wd:Q25917154,  wd:Q1243306, wd:Q1570262, wd:Q811683, wd:Q744913          ) )
+            }
+            GROUP BY ?place ?placeLabel
+            """
+
+_PITTSBURGH_QUERY = """SELECT ?place ?placeLabel
+                    (GROUP_CONCAT(DISTINCT?location;separator=", ") AS ?point)
+                    WHERE 
+                    {
+                    {
+                        ?place wdt:P31 ?instance.
+                        SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+                        SERVICE wikibase:box {
+                        ?place wdt:P625 ?location .
+                        bd:serviceParam wikibase:cornerWest "Point(-80.035,40.425)"^^geo:wktLiteral .
+                        bd:serviceParam wikibase:cornerEast "Point(-79.930,40.460)"^^geo:wktLiteral .
+                        }
+                    }
+                    FILTER (?instance  not in
+                    (wd:Q34442,wd:Q12042110,wd:Q124757,wd:Q79007,wd:Q18340514,wd:Q537127,wd:Q1311958,wd:Q124757,
+                    wd:Q25917154,  wd:Q1243306, wd:Q1570262, wd:Q811683,
+                    wd:Q744913          ) )
+                    }
+                    GROUP BY ?place ?placeLabel
+                """
+
+
+def get_geofenced_wikidata_items(region: Text) -> Dict:
+    '''Get Wikidata items for a specific area. 
+    Arguments:
+        region(Text): The area to query the Wikidata items.
+    Returns:
+        The Wikidata items found in the area.
+    '''
+
+    assert region == "Manhattan" or region == "Pittsburgh"
```

Comment:
Having an assert here means that the 'else' part on L176 will never be reached.
Instead, you can have the error message in the assert itself 
e.g. :
assert region == "Manhattan" or region == "Pittsburgh", "{region} not supported."

Reply:
forgot to remove the assert

Reply:
you are right, I forgot to remove this line, did that now.